src/main/java/seedu/address/model/event/Event.java

```diff
@@ -20,20 +20,20 @@
     private final EventVenue venue;
     private final EventManpowerNeeded manpowerNeeded;
     private final EventHoursNeeded hoursNeeded;
-    private final EventStartDate startDate;
-    private final EventEndDate endDate;
+    //private final EventStartDate startDate;
+    //private final EventEndDate endDate;
     private final EventManpowerAllocatedList manpowerAllocatedList;
     private final Set<Tag> tags = new HashSet<>();
 
+    //public Event(EventId id, EventName name, EventVenue venue, EventHoursNeeded hoursNeeded,
+    //EventManpowerNeeded manpowerNeeded, EventStartDate startDate, EventEndDate endDate, Set<Tag> tags) {
     public Event(EventId id, EventName name, EventVenue venue, EventHoursNeeded hoursNeeded,
```

Comment:
eventId can be static variable and generated by system to avoid duplication

Reply:
@calvincxz The static variable is encapsulated inside the EventId class itself